app/src/main/java/com/framgia/englishconversation/data/source/remote/timeline/TimelineRemoteDataSource.java

```diff
@@ -174,11 +173,13 @@ public void onCancelled(DatabaseError databaseError) {
     }
 
     @Override
-    public Observable<TimelineModel> registerModifyTimeline(final TimelineModel lastTimeline,
-                                                            final UserModel userModel) {
-        return Observable.create(new ObservableOnSubscribe<TimelineModel>() {
+    public Observable<GenericsModel<Integer, TimelineModel>> registerModifyTimeline
+            (final TimelineModel lastTimeline, final UserModel userModel) {
```

Comment:
ERROR: 'method def lparen' have incorrect indentation level 12, expected level should be 4.